src/Types.hs

```diff
@@ -285,12 +285,21 @@ type instance Proto IdeaLike = ()
 data IdeaVote = IdeaVote
     { _ideaVoteMeta  :: MetaInfo IdeaVote
     , _ideaVoteValue :: IdeaVoteValue
+    , _ideaVoteVoter :: AUID User
```

Comment:
rename to `_ideaVoteDelegate`?  also document which of delegate and delegator is stored in meta info.